kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -17,27 +19,24 @@ class APIDataSet(AbstractDataSet[None, requests.Response]):
     """``APIDataSet`` loads the data from HTTP(S) APIs.
     It uses the python requests library: https://requests.readthedocs.io/en/latest/
 
-    Example usage for the
-    `YAML API <https://kedro.readthedocs.io/en/stable/data/\
+    Example usage for the `YAML API <https://kedro.readthedocs.io/en/stable/data/\
     data_catalog.html#use-the-data-catalog-with-the-yaml-api>`_:
 
     .. code-block:: yaml
 
         usda:
           type: api.APIDataSet
           url: https://quickstats.nass.usda.gov
-          params:
+        params:
```

Comment:
I still think this is wrong? Did I misunderstand something

Reply:
my editor keeps messing this line